src/Schema/Events/Lists.ts

```diff
@@ -0,0 +1,115 @@
+import { OwnerType, PageOwnerType } from "../Values/OwnerType"
+import { ActionType } from "."
+
+/**
+ * Schema describing 'Lists' events
+ * @packageDocumentation
+ */
+
+/**
+ * When a user adds an artwork to an artwork list
+ *
+ * This schema describes events sent to Segment from [[addedArtworkToArtworkList]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "addedArtworkToArtworkList",
```

Comment:
**question**:  do we need to use `ArtworkList` in events or just `List` (e.g. `addedArtworkToList`). Based on [this Notion Card](https://www.notion.so/artsy/Artworks-list-I-think-we-ve-got-a-confusing-mix-of-terms-in-our-codebase-for-Artwork-Lists-collec-03de616986ab4cd4a839ec6aed989d3d)

cc: @anandaroop @nickskalkin

Reply:
[Discussed and decided](https://artsy.slack.com/archives/C9SATFLUU/p1680012458469089?thread_ts=1680012130.077609&cid=C9SATFLUU) to use` ArtworkList` term